packages/shared/src/components/cards/PostMetadata.tsx

```diff
@@ -18,24 +22,35 @@ export default function PostMetadata({
   className,
   children,
   description,
+  isVideoType,
+  insaneMode,
 }: PostMetadataProps): ReactElement {
   const date = useMemo(
     () => createdAt && postDateFormat(createdAt),
     [createdAt],
   );
 
+  const timeActionContent = isVideoType ? 'watch' : 'read';
+
   return (
     <div
       className={classNames(
         'flex items-center text-theme-label-tertiary typo-footnote',
         className,
       )}
     >
+      {isVideoType && insaneMode && (
```

Comment:
Icon here is smaller then the design, but the XSmall version is bigger, so checking this with design.

Reply:
Confirmed with design for XXSmall